### DISCUSSION TITLE
Domain name does not end with a valid public suffix (TLD)

### DISCUSSION THREAD

**Discussion by TomAndThat** [dokku/dokku#5443] [Q&A] [answered]
Hi folks,

I hope this is the right place to ask this question - apologies if not.

I'm trying to install an SSL cert with dokku-letsencrypt, but I'm getting the below error:

```
2022/11/04 14:31:23 Could not obtain certificates:
        acme: error: 400 :: POST :: https://acme-v02.api.letsencrypt.org/acme/new-order :: urn:ietf:params:acme:error:rejectedIdentifier :: Error creating new order :: Cannot issue for "liz-truss-game.liz-truss-game": Domain name does not end with a valid public suffix (TLD)
```

liz-truss-game is the name of the app, but is obviously not the domain (which is liztrussgame.com); however, I'm not sure where I input the domain name. Is there a stage that I've missed to set the domain name as a variable, for example?

The steps I've taken so far are to install the plugin:

`dokku plugin:install https://github.com/dokku/dokku-letsencrypt.git`

And then to set my email: 

`dokku config:set --global DOKKU_LETSENCRYPT_EMAIL=[my email]`

Is there a step that I've missed between this and `dokku letsencrypt:enable liz-truss-game`

**Comment by josegonzalez** [comment, marked as answer]
This is probably due to you having a hostname of `liz-truss-game`, resulting in that being treated as the global domain name on install.

You can show your current domains via `dokku domains:report $APP` and remove any that are invalid via `dokku domains:remove $APP $DOMAIN`.

**Reply by TomAndThat** [reply]
Yep - sorted it. Thanks so much for your help! Really appreciate it.
